### DISCUSSION TITLE
Disable Auto Update Feature

### DISCUSSION THREAD

**Discussion by Lennox-dev** [Devolutions/UniGetUI#2918] [Q&A]
I was wondering is there an option to disable the app from auto updating\installing packages and running on it's own? I like it to run entirely manually, whenever I want

**Comment by marticliment** [comment]
- Automatic package updates are disabled by default
 - You can disable autostart from task manager or from Windows Settings
 - You can disable the self-updater from UniGetUI Settings

**Comment by Lennox-dev** [comment]
I might be wrong, but I experienced an unusual activity that packages updating feature were going on even it was disabled inside app, and app was not even running (Icon was not visible in the taskbar). Does app add itself to Windows task scheduler to run itself automatically in background?

**Reply by marticliment** [reply]
No, if the app was not showing on the taskbar then it was not running.
Please take into account that some apps autoupdate themselves, and then the changes get reflected into UniGetUI, but it wasn't UniGetUI who updated those apps

**Reply by Lennox-dev** [reply]
Thanks!
